include/deal.II/base/config.h.in

```diff
@@ -267,6 +267,23 @@
 
 
 /***********************************************************************
+ * A macro that is used to mark function as not throwing any exceptions.
+ * The appropriate way to do this in C++98 was to say
+ *     void foo () throw ();
```

Comment:
Is this right? I thought this sort of annotation said that a function *could* throw an exception, which would not be equivalent to `noexcept` (which cannot throw).